### DISCUSSION TITLE
Welcome to apps Discussions!

### DISCUSSION THREAD

**Discussion by Raphty** [portmaster-community/apps#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jmeverett** [comment]
Hello everyone,  I am MikeJ. I promote Portmaster in the US, help with marketing campaigns and moderate the Discord. I look forward to conversing with all of you about Portmaster.

**Reply by Nexxman** [reply]
Hi Mike, new to Portmaster. Look forward to learning a lot more about it. .

**Reply by jmeverett** [reply]
Stop by our Discord, great conversation, tips, and ideas along with a great community of people!

**Reply by Raphty** [reply]
Welcome @Nexxman
